core/src/services/cloudflare_kv/backend.rs

```diff
@@ -292,7 +293,8 @@ pub struct CfKvResponse {
 #[derive(Debug, Deserialize)]
 pub struct CfKvScanResponse {
     pub errors: Vec<CfKvError>,
-    pub messages: Vec<CfKvMessage>,
+	#[allow(dead_code)]
```

Comment:
Please remove this field (or comment) instead of marking as `dead_code`.

Reply:
sorry，i will try it tomorrow.

Reply:
Have a great weekend!

Reply:
I commentted lots of unused fields. But I don't know is it ok?

Reply:
> I commentted lots of unused fields. But I don't know is it ok?

Will it be better to rename unused field to _xx and then add #[serde(rename("xx")] ?